### PR TITLE
generate docsets

### PR DESCRIPTION
**Pull request by mayel**
Hi, I know this was an old repo, but in case you're still interested, I haven't solved the feed not being recognised by Zeal, but made it work another way :)

### CODE REVIEW

File: lib/file_ext.ex [hunk 1]
```diff
@@ -1,13 +1,22 @@
 defmodule FileExt do
   def ls_r(path \\ ".") do
     cond do
-      File.regular?(path) -> [path]
+      File.regular?(path) ->
+        # case Path.extname(path) do
+        #   ".html" -> [path]
+        #   _ -> []
+        # end
```
Comment (johnhamelink) [inline lib/file_ext.ex]: Can you remove this dead code?

File: mix.exs [hunk 2]
```diff
@@ -2,46 +2,56 @@ defmodule DocsetApi.Mixfile do
...
-     elixirc_paths: elixirc_paths(Mix.env),
-     compilers: [:phoenix, :gettext] ++ Mix.compilers,
-     build_embedded: Mix.env == :prod,
-     start_permanent: Mix.env == :prod,
-     aliases: aliases(),
-     deps: deps()]
+    [
+      app: :docset_api,
+      version: "0.1.0",
+      elixir: "~> 1.2",
```
Comment (johnhamelink) [inline mix.exs]: Did you use elixir ~> 1.2 to make and test these changes? 1.10 is the current latest release

**Review by johnhamelink — CHANGES_REQUESTED** [review]

**Comment by johnhamelink** [comment]
Hi @mayel thanks for this contribution, it looks like you've fixed something here! I'm going to test it, but in the meantime could you let me know which version of Elixir you built this with? As you rightly pointed out, this repo hasn't seen much love in quite a while :)

**Comment by mayel** [comment]
I was working with Elixir 1.10.3, and upgraded the dependencies to more recent ones.

BTW regarding feeds, I solved the problem you were up against (unknown protocol error) by putting a full URL in the feed instead of a relative one, but now Zeal fails silently when trying to add the feed.

**Comment by johnhamelink** [comment]
@mayel I'm going to test this PR against 1.10.4 and Erlang 23.0.3 (using asdf to install), I'll take a look at the error you mention as well and comment my findings on this PR. Thanks :100:

**Comment by mayel** [comment]
I've made some updates but can't find how to add a commit to the existing PR :/

**Comment by johnhamelink** [comment]
@mayel if you like I can help you over the Elixir slack channel or IRC :) Let me know

**Comment by mayel** [comment]
Cool, I'm more used to Gitab.
 @johnhamelink I'm on both (same username on Slack)

**Comment by mayel** [comment]
Oh it looks like the commit was added automatically, but it was just the UI that confused me.

**Review by johnhamelink — APPROVED** [review]

**Pull request merged**